src/lib/Quote.svelte

```diff
@@ -17,6 +17,7 @@
     }
 
     blockquote {
+        position: relative;
```

Comment:
This way of positioning the quotes looks kinda wonky on larger screens but it definitedly looks better than the previous iteration!